pages/common/mongo.md

```diff
@@ -0,0 +1,19 @@
+# mongo
+
+> MongoDB shell client.
+
+- Connect to a database:
+
+`mongo {{database}}`
+
+- Connect to a database running on a given host on a given port:
+
+`mongo --host {{host}} --port {{port}} {{database}}`
+
+- Connect to a database with a given username, user will be prompted for password:
+
+`mongo --username {{username}} {{database}} --password`
+
+- Run queries on the database:
```

Comment:
Not really. You can actually evaluate complete javascript expressions which can run multiple queries. I agree `query` might be a bit misleading here. I thought of `javascript`, but that is too vague.

Open to other options here.